### ISSUE TITLE
Is specimen mandatory?

### ISSUE THREAD

**Issue by FrederikVds** [hl7-be/lab#211]
We're trying testcase 1B and get the following result:

```
{
    fhirpath: [
        {
             actual: false,
             field: "Bundle.entry.where(resource is Observation).resource.specimen.distinct().count() > 2",
             expected: true
        },
        {
            actual: "Error during execution of rule",
            field: "Bundle.entry.resource.meta.profile.aggregate($total & $this).contains('www.ehealth.fgov.be/standards/fhir/S').not()",
            expected: true
        }
    ]
}
```

The first issue is that we don't include specimen information in our reports. This does not seem mandatory in the FHIR standard. Am I missing something?

I also don't understand the second issue. What meta information is expected?

**Comment by bdc-ehealth** [comment]
@FrederikVds ,

Hi Frederik, Specimen is not mandatory, but for this specific test, there are 3 glucose measurements or CRP mesurements needed in three different specimen. That is why three specimens are required here. 

The meta information required here is the name of the profile that you are using: in XML, this would be something like this:

```
<meta>
                    <profile value="https://www.ehealth.fgov.be/standards/fhir/lab/StructureDefinition/be-laboratory-report-composition"/>
                </meta>
```
Files without this information do not show up in the visualizer: https://fhir-testserver.be/index.php/visualization_webapp . They can be valid FHIR, but if they do not refer to the Belgian profiles, the validation software cannot guess it should apply the Belgian validation rules.

**Comment by bdc-ehealth** [comment]
@FrederikVds ,

Yes, you can, but only if the LOINC / ReTaM codes are unambiguous about the Specimen (i.e. the type of specimen occurs in the definition of the ReTaM code).
